admin/ownership_grants/V1.6.3__internamespace_object_ownership_prod.sql

```diff
@@ -0,0 +1,19 @@
+-- Grant ownership of internamespace objects to proxy admin database role
+-- SYNAPSE
+GRANT OWNERSHIP
+	ON ALL DYNAMIC TABLES
+	IN SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE
+	TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.ALL_ADMIN
+	COPY CURRENT GRANTS;
+GRANT OWNERSHIP
+	ON ALL TASKS
+	IN SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE
```

Comment:
We don't have task in `synapse_data_warehouse.synapse`

Reply:
We have `BACKUP_SYNAPSE_DATA_WAREHOUSE_TASK`, which is owned by SYSADMIN. But now that I think of it, this task needs the CREATE DATABASE privilege, which we don't want to grant to the proxy admin role since it will be cloned and granted to developers. I think I'll remove this line.